MoviesApp/MoviesApp/FirstScreen/Presenter/FilmsTableViewPresentable.swift

```diff
@@ -0,0 +1,12 @@
+// FilmsTableViewPresentable.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Presenter protocol
```

Comment:
```suggestion
/// Presenter protocol
```
Коменты или все на русском или все на англ